tests/testthat/test-values.R

```diff
@@ -84,9 +84,12 @@ test_that("sequences - doubles", {
   expect_equal(
     value_seq(mixture(), 1), 0
   )
-  expect_equal(
-    value_seq(test_param_3, 1), .40
+  expect_snapshot(
+    value_seq(test_param_3, 1)
   )
+
+  withr::local_options(lifecycle_verbosity = "quiet")
```

Comment:
We think you won't need to touch this global option at all once all the other changes have been made